packages/frames.js/src/getFrameFlattened.ts

```diff
@@ -1,3 +1,4 @@
+import { version as framesjsVersion } from "../package.json";
```

Comment:
I'm not sure this will works. I also commented on Linear about this. This purely depends on fact WHEN the `version` property is bumped in `package.json`. If it is bumped BEFORE build step, then this will work, if it is bumped AFTER build step, then it will reflect the previous version instead of new one.

This import is inlined by `tsup`.

Reply:
@michalkvasnicak i added a rebuild of the frames.js package after the version bumping in the build command. let me know what you think of this solution